civihr_default_theme/scss/components/_leave-report.scss

```diff
@@ -0,0 +1,56 @@
+.chr_leave-report__action {
+  color: $gray-darker;
+}
+
+
+.chr_leave-report__label {
+  color: $gray-darker;
+  font-weight: 600;
+
+  > .fa {
+    margin-right: 5px;
+  }
+}
+
+.chr_leave-report__label--toggle {
+  cursor: pointer;
+  user-select: none;
+}
+
+
+.chr_leave-report__heading {
+
+  .chr_leave-report__export-button {
+    line-height: 2.4;
```

Comment:
i don't think it makes sense to have it as a variable. It's a specific one-time value used only for this element